validator/java/install_bazel.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+# This script installs Bazel, either by downloading it or by installing from the
+# Travis cache if available.
+CYAN() { echo -e "\033[0;36m$1\033[0m"; }
+YELLOW() { echo -e "\033[1;33m$1\033[0m"; }
+GREEN() { echo -e "\033[0;32m$1\033[0m"; }
+RED() { echo -e "\033[0;31m$1\033[0m"; }
+
+LOG_PREFIX=$(YELLOW "install_bazel.sh")
+BAZEL_VERSION="2.2.0"
+BAZEL_BIN_URL="https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel_$BAZEL_VERSION-linux-x86_64.deb"
+INSTALLER_DIR="bazel-installer"
+BAZEL_BIN_PATH="$INSTALLER_DIR/bazel_$BAZEL_VERSION-linux-x86_64.deb"
```

Comment:
As written, this will only work on linux. If there is value in making this work on MacOS, we can add OS detection logic.

https://github.com/ampproject/amphtml/blob/06b6ed4d1e59df8697994b5473d223fae9504c14/build-system/sauce_connect/start_sauce_connect.sh#L41-L52

Since you don't have a Macbook, I'm happy to do this in a follow up 😃

Reply:
I saw that logic there, and at the time was thinking it was targeted toward Travis on MacOS, which we don't do. I suppose it's also needed for Mac devs to install bazel, though. My inclination is that this should be a pretty trivial addition, but you're correct that I don't have a Macbook or any sane way to validate. So, I'd support punting the Darwin logic to you.

I added in a partial implementation for you to piggy-back off of, currently it will abort on anything but linux.

Reply:
Done in #27706.